src/controllers/v1.controller.ts

```diff
@@ -19,6 +19,8 @@ import { loadBlockchain } from '../utils/relayer'
 import { SendRelayOptions } from '../utils/types'
 const logger = require('../services/logger')
 
+const DEFAULT_STICKINESS_DURATION = 300
```

Comment:
Is this in seconds?

Reply:
As it is a redis key TTL, yes it is. A comment here behind it `// in seconds` would help.